### PR TITLE
zh: update /zh/docs/concepts/scheduling/kube-scheduler.md

### PR DESCRIPTION
**Pull request by yuxiaobo96**
Add `zh` to the link in the document to make it jump to the Chinese page.
And fix a translation error.
ref: #18593

### CODE REVIEW

File: content/zh/docs/concepts/scheduling/kube-scheduler.md [hunk 1]
```diff
@@ -57,7 +57,7 @@ is the default scheduler for Kubernetes and runs as part of the
 kube-scheduler is designed so that, if you want and need to, you can
 write your own scheduling component and use that instead.
 -->
-[kube-scheduler](https://kubernetes.io/docs/reference/command-line-tools-reference/kube-scheduler/) 是 Kubernetes 集群的默认调度器，并且是集群 {{< glossary_tooltip text="控制面" term_id="control-plane" >}} 的一部分。如果你真的希望或者有这方面的需求，kube-scheduler 在设计上是允许你自己写一个调度组件并替换原有的 kube-scheduler。
+[kube-scheduler](https://kubernetes.io/zh/docs/reference/command-line-tools-reference/kube-scheduler/) 是 Kubernetes 集群的默认调度器，并且是集群 {{< glossary_tooltip text="控制面" term_id="control-plane" >}} 的一部分。如果你真的希望或者有这方面的需求，kube-scheduler 在设计上是允许你自己写一个调度组件并替换原有的 kube-scheduler。
```
Comment (gavinfish) [inline content/zh/docs/concepts/scheduling/kube-scheduler.md]: Why do these links need prefix `https://kubernetes.io`?

Comment (yuxiaobo96) [reply content/zh/docs/concepts/scheduling/kube-scheduler.md]: I think Chinese translated articles should jump to the associated Chinese page.

Comment (gavinfish) [reply content/zh/docs/concepts/scheduling/kube-scheduler.md]: I mean `[kube-scheduler](/zh/docs/reference/command-line-tools-reference/kube-scheduler/)` should also work, right? I can see both patterns across different files.

Comment (yuxiaobo96) [reply content/zh/docs/concepts/scheduling/kube-scheduler.md]: 或许我误解你的意思了，`https://kubernetes.io`前缀不是我加的，是英文原文中自带的

Comment (sftim) [reply content/zh/docs/concepts/scheduling/kube-scheduler.md]: I agree with @gavinfish; to follow SIG Docs recommendations, this should be:
```suggestion
[kube-scheduler](/zh/docs/reference/command-line-tools-reference/kube-scheduler/) 是 Kubernetes 集群的默认调度器，并且是集群 {{< glossary_tooltip text="控制面" term_id="control-plane" >}} 的一部分。如果你真的希望或者有这方面的需求，kube-scheduler 在设计上是允许你自己写一个调度组件并替换原有的 kube-scheduler。
```
(one day, we'll have a linter for this. one day.)

File: content/zh/docs/concepts/scheduling/kube-scheduler.md [hunk 2]
```diff
@@ -304,16 +304,16 @@ kube-scheduler 有一系列的默认调度策略。
   没有归属于同一个 Service 的 Pod，这个策略更倾向于将 Pod 调度到这类 Node 上。最终的目的：即使在一个 Node 宕机之后 Service 也具有很强容灾能力。
 
 - `CalculateAntiAffinityPriorityMap`：这个策略主要是用来实现[pod反亲和]
-  (https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#affinity-and-anti-affinity)。
-  
+  (https://kubernetes.io/zh/docs/concepts/configuration/assign-pod-node/#affinity-and-anti-affinity)。
```
Comment (sftim) [inline content/zh/docs/concepts/scheduling/kube-scheduler.md]: May as well fix this to:
```suggestion
  (/zh/docs/concepts/configuration/assign-pod-node/#affinity-and-anti-affinity)。
```

**Comment by sftim** [comment]
It would be nice to take out the `https://kubernetes.io` but that _can_ happen in a different PR. This PR is not making things **worse**.

File: content/zh/docs/concepts/scheduling/kube-scheduler.md (hunk 1 above)
Comment (yuxiaobo96) [reply content/zh/docs/concepts/scheduling/kube-scheduler.md]: done

Comment (yuxiaobo96) [reply content/zh/docs/concepts/scheduling/kube-scheduler.md]: Thank you both for your suggestions

File: content/zh/docs/concepts/scheduling/kube-scheduler.md (hunk 2 above)
Comment (yuxiaobo96) [reply content/zh/docs/concepts/scheduling/kube-scheduler.md]: done

**Pull request merged**